### DISCUSSION TITLE
Add-PnPSiteCollectionAdmin

### DISCUSSION THREAD

**Discussion by m3d0c** [pnp/powershell#294] [Q&A] [answered]
This is the first time I have used this command after switching to the new PowerShell command set and in the old version this command worked to allow add additional users as site collections admin

In the new version I get "WARNING: Unable to add user as it wasn't found. User will be skipped." I have manually tried to add 3 different accounts and they all have this same message. 

Any ideas

**Comment by LeonArmston** [comment, marked as answer]
> This is the first time I have used this command after switching to the new PowerShell command set and in the old version this command worked to allow add additional users as site collections admin
> 
> In the new version I get "WARNING: Unable to add user as it wasn't found. User will be skipped." I have manually tried to add 3 different accounts and they all have this same message.
> 
> Any ideas

Hi

This looks very similar to a very recent issue https://github.com/pnp/powershell/issues/291 and your issue may be that the users dont already have access to the site?

A fix has been made and should be available in tonight's nightly release for this.

Thanks
